Documentation/ExtensionArchitecture/DeclarationFile/Index.rst

```diff
@@ -20,6 +20,9 @@ The keys are described in the table below.
 This file is overwritten, when extensions are imported from the online repository. So don't write your custom code in this file - only change
 values in the :php:`$EM_CONF` array if needed.
 
+.. important::
+However, due to limitations to TER, the `$_EXTKEY` option should be kept within an extension’s ext_emconf.php. Do **not** use a constant or a string.
```

Comment:
$_EXTKEY is not mentioned on that page at all. So if we mention that, it might be a good idea to add a small code snippet or something to clarify what we mean.

(I realize this is the same text as on the other page, but on the other page there is a little more context). 

I suggest to add a complete example for contents of ext_emconf.php. This might be helpful anyway.